core/web/router.go

```diff
@@ -332,14 +332,6 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.POST("/jobs", jc.Create)
 		authv2.DELETE("/jobs/:ID", jc.Delete)
 
-		jpc := JobProposalsController{app}
```

Comment:
We can safely remove this since it was only ever used by the UI which has since migrated to use GQL